dotcom-rendering/src/web/components/FixedLargeSlowXIV.tsx

```diff
@@ -95,61 +87,99 @@ export const FixedLargeSlowXIV = ({
-							return (
-								<LI
-									padSides={true}
-									percentage="25%"
-									showDivider={cardIndex !== 0}
-								>
-									<Card
-										containerPalette={containerPalette}
-										showAge={showAge}
-										linkTo={card.url}
-										format={card.format}
-										headlineText={card.headline}
-										headlineSize="small"
-										imageUrl={
-											groupIndex === 0
-												? card.image
-												: undefined
-										}
-										imagePosition="top"
-										byline={card.byline}
-										showByline={card.showByline}
-										showQuotes={
-											card.format.design ===
-												ArticleDesign.Comment ||
-											card.format.design ===
-												ArticleDesign.Letter
-										}
-										webPublicationDate={
-											card.webPublicationDate
-										}
-										kickerText={card.kickerText}
-										showPulsingDot={
-											card.format.design ===
-											ArticleDesign.LiveBlog
-										}
-										showSlash={true}
-										showClock={false}
-										mediaType={card.mediaType}
-										mediaDuration={card.mediaDuration}
-										starRating={card.starRating}
-										branding={card.branding}
-										discussionId={card.discussionId}
-										dataLinkName={card.dataLinkName}
-										snapData={card.snapData}
-									/>
-								</LI>
-							);
-						})}
-					</UL>
-				);
-			})}
+			<UL direction="row" padBottom={true}>
+				{secondSlice.map((card, cardIndex) => {
+					return (
+						<LI
+							padSides={true}
+							percentage="25%"
+							showDivider={cardIndex !== 0}
+							key={card.url}
+							padBottomOnMobile={cardIndex != 3}
+						>
+							<Card
+								containerPalette={containerPalette}
+								showAge={showAge}
+								linkTo={card.url}
+								format={card.format}
+								headlineText={card.headline}
+								headlineSize="small"
+								imageUrl={card.image}
+								imagePosition="top"
+								byline={card.byline}
+								showByline={card.showByline}
+								showQuotes={
+									card.format.design ===
+										ArticleDesign.Comment ||
+									card.format.design === ArticleDesign.Letter
+								}
+								webPublicationDate={card.webPublicationDate}
+								kickerText={card.kickerText}
+								showPulsingDot={
+									card.format.design ===
+									ArticleDesign.LiveBlog
+								}
+								showSlash={true}
+								showClock={false}
+								mediaType={card.mediaType}
+								mediaDuration={card.mediaDuration}
+								starRating={card.starRating}
+								branding={card.branding}
+								discussionId={card.discussionId}
+								dataLinkName={card.dataLinkName}
+								snapData={card.snapData}
+							/>
+						</LI>
+					);
+				})}
+			</UL>
+			<UL direction="row" padBottom={true} wrapCards={true}>
+				{thirdSlice.map((card, cardIndex) => {
+					return (
+						<LI
+							padSides={true}
+							percentage="25%"
+							showDivider={cardIndex % 4 !== 0}
```

Comment:
```suggestion
							showDivider={cardIndex !== 4 && cardIndex !== 8}
```

mod is neat but it's niche, it might be more inclusive to be more explicit, albeit more verbose

Reply:
good call, thanks! I partly had my mind still on the show more case, where we need something similar but without knowing how many rows there will be. I think you're right though that spelling the conditions out explicitly here is good, and if I use modulo in the more flexible version then I will wrap it in a descriptive function that makes clear what it's doing 👍